### ISSUE TITLE
Top-level Makefile

### ISSUE THREAD

**Issue by binkley** [brianduff/mlogagent#1]
This is a nifty project!  I'm a "printf"/"println" kind of programmer for debugging, and appreciate what you're doing here.

It would be handy to provide a top-level `Makefile` with `README.md` instructions.  Ideally, I'd like to clone the project, look at the README, and just run:
```
$ make
```

**Comment by brianduff** [comment]
Yep, definitely agree & thanks for the feedback. Most of the things I tinker here with use bazel, but this one is simple enough that a makefile makes a lot of sense. I'll send up a commit for that.
